www/src/views/showcase/showcase-list.js

```diff
@@ -104,6 +104,13 @@ const ShowcaseList = ({ items, count }) => {
             </div>
           )
       )}
+      {/* makes last row items equal width and aligned left */}
```

Comment:
Ries, this is such a big improvement for those pages! Thanks so much!

Am I right remembering the "empty item" hack is the only way to do this kind of layout with flexbox?

Reply:
yeah, to make it look like this that's the only solution I could find

Reply:
this is what it looks like if you leave it out because of the min/max width of the items
![screen shot 2018-09-09 at 00 36 51](https://user-images.githubusercontent.com/36336875/45259256-87531400-b3c8-11e8-9e30-b9ed1580f31e.png)

Reply:
How do you feel about using CSS grids for this btw.?
I'm a total noob still, https://gridbyexample.com/ blows my mind every time I visit 🤦‍♂️ 😸…browser support looks good!

Reply:
I thought about this quite a bit and personally 👍getting this in with flexbox and the empty elements to facilitate handling the items in the last row as long as we make sure they don't cause any accessibility issues.

I think they shouldn't as we e.g. don't render our list as an unordered list, where those items would be empty `<li>`s, which I think would be considered by screenreaders IIRC. Need to brush up my knowledge here, too, though—any input here is highly appreciated!

As long as those empty elements don't cause any harm, getting this in gives us more time to think about implementation details while we improve the UX and presentation of sites and starters, so 👍 again!

Reply:
if we add the "aria-hidden" attribute to the elements they shouldn't get picked up by screen readers...

Reply:
Right, perfect!

Reply:
so how does this work, do I create a new pr with the changes? or can I make changes to this one? not sure how to do it...

Reply:
@riencoertjens 👋 

Largely, it depends. I'd say if the changes make sense to make their way into this PR (i.e. something is broken, or it's worth fixing it _before_ merging) then let's withhold merging and commit fixes/changes to this specific PR.

On the other hand, if it's more of an enhancement request that could be done in a future PR (and there's value in getting _this_ PR merged first) then that is sometimes a good reason to just merge and fix in a later PR.

For this one, I'd recommend addressing the feedback in _this_ PR and we can iterate further on this before merging it in. Sound like a plan?